### PR TITLE
add search bars

### PR DESCRIPTION
**Pull request by manch0ffline**
- [DEMO LINK](https://manch0ffline.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://manch0ffline.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html
Comment (volodymyr-soltys97) [inline src/index.html:14]: Use a semantic tag for for inputs instead of main

File: src/index.html
Comment (volodymyr-soltys97) [inline src/index.html:19]:
```suggestion
        <img
          src="images/Search.svg"
          alt="search"
          class="img img--big"
        >
```

File: src/index.html
Comment (volodymyr-soltys97) [inline src/index.html:32]:
```suggestion
        <img
          src="images/Search.svg"
          alt="search"
          class="img img--small"
        >
```

File: src/index.html
Comment (volodymyr-soltys97) [inline src/index.html:19]: Checklist, you need to use image as a background property
image

File: src/style.css
Comment (volodymyr-soltys97) [inline src/style.css:9]: Default value, so you can just remove it everywhere
```suggestion
```

File: src/style.css
Comment (volodymyr-soltys97) [inline src/style.css:27]: Create a variable with root selector

File: src/style.css
Comment (volodymyr-soltys97) [inline src/style.css:33]: Remove comments
```suggestion
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code

File: src/style.css
Comment (maxim2310) [inline src/style.css:9]: also need to add font weight

File: src/style.css
Comment (maxim2310) [reply src/style.css:26]: not fixed https://github.com/mate-academy/layout_search-bar-airbnb/pull/3817/files#r1407405161

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Great job overall! Just couple tiny things needs to be fixed before approval)

**Pull request closed without merging**
